### DISCUSSION TITLE
Album ART and song titles

### DISCUSSION THREAD

**Discussion by jtome7** [smplayer-dev/smplayer#439] [General]
Sometimes I see the Album ART and sometimes I do not.  Can that feature become the DEFAULT to show album art and song titles?

**Comment by smplayer-dev** [comment]
It's the default but not all songs include a cover image.

**Reply by a-pav** [reply]
Is there anyway to disable the showing of cover art? I tried passing [`--audio-display=no`](https://manpages.debian.org/bookworm/mpv/mpv.1.en.html#no~4) option in Advanced > MPlayer/mpv but it didn't make any difference, the cover art is still showing.

Edits:

- I noticed that `Hide video window when playing audio files` in Interface tab is also enabled.

- It appears to be caching matter. Once a file is loaded with its cover art, doesn't matter if you go and add `--audio-display=no` later, it keeps loading the cover art unless the caches at `~/.config/smplayer/file_settings` are removed.

**Reply by buzzcut121** [reply]
I also have this problem. I downloaded a song that had album art and it appeared when I play that song: Not only that, but that same album art is also appearing on every single song in the same directory. Where is SMPlayer pulling the album art from? How can I clear the cache? I went and deleted the album art from the song file itself using mp3tag and saved it. The album art is now gone from the file itself (viewing through mp3tag) but SMPlayer is still displaying that album art for every song. 
I moved the song file to a different directory, played it... No album art. Moved it back to the original directory & played... the album art appeared again. 

I am running player 23.12 on Windows 10 home.
